extensions/juejin/package.json

```
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "jue-jin",
+  "title": "juejin",
```

```suggestion
  "title": "Juejin",
```

Let's use the title case `Juejin` for its display name.

---

okay, I also think this is better.